front/app/components/UI/LocationInput/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import AsyncSelect from 'react-select/async';
+import selectStyles from 'components/UI/MultipleSelect/styles';
+import fetcher from 'utils/cl-react-query/fetcher';
+import useLocale from 'hooks/useLocale';
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+export interface Option {
+  label: string;
+  value: string;
+}
+
+type TextSearchResponse = {
+  data: {
+    type: string;
+    attributes: {
+      results:
+        | {
+            formatted_address: string;
+          }[]
+        | undefined;
+    };
+  };
+};
+
+const LocationInput = (props: React.ComponentProps<typeof AsyncSelect>) => {
+  const { formatMessage } = useIntl();
+  const locale = useLocale();
+  const promiseOptions = async (inputValue: string) => {
+    try {
+      const response = await fetcher<TextSearchResponse>({
+        path: '/location/autocomplete',
```

Comment:
Could this request be in `locationTools.ts` together with two others?

Reply:
I think this request is very specific to the location input (not used elsewhere) so it's best if we keep it co-located.